internal/rooms/biomes.go

```diff
@@ -13,6 +22,21 @@ type BiomeInfo struct {
 	burns          bool // Does this area catch fire? (brush etc.)
 }
 
+type BiomeConfig struct {
+	Name           string `yaml:"name"`
+	Symbol         string `yaml:"symbol"`
+	Description    string `yaml:"description"`
+	DarkArea       bool   `yaml:"darkArea"`
+	LitArea        bool   `yaml:"litArea"`
+	RequiredItemId int    `yaml:"requiredItemId"`
+	UsesItem       bool   `yaml:"usesItem"`
+	Burns          bool   `yaml:"burns"`
+}
```

Comment:
all yaml properties and yaml:"name" should be lowercase... camelCase etc. should be avoided.

The reason for this originally is that it was the default convention in the yaml library, but it also helps to not have to guess/remember the uppercase letters when editing yaml.

Reply:
Of course the struct fields can be cased however needed. Just the yaml form needs to change.